builds/misc/packages-release.yaml

```diff
@@ -229,6 +232,30 @@ stages:
             PACKAGE_FILTER: $(identityServicePackageFilter)
             DOWNLOAD_PATH: $(System.ArtifactsDirectory)/$(artifactName)
             IDENTITY_SERVICE_COMMIT: $(aziotis.commit)
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+          displayName: ESRP Binary CodeSigning (IIS)
+          inputs:
+            ConnectedServiceName: "Azure IoT Edge Code Sign 2"
+            FolderPath: '$(System.ArtifactsDirectory)/$(artifactName)'
```

Comment:
Should this not include the '/$(artifactName)'?

Reply:
I think it should since each of the `job` will be its own specific OS artifact -- all the OSes share the same `$(System.ArtifactsDirectory)` which will cause the directory to be published multiple times.